scripts/configure-installer-tests-cluster.sh

```diff
@@ -11,12 +11,35 @@ SETUP_OPERATORS="./scripts/setup-operators.sh"
 DEFAULT_INSTALLER_ASSETS_DIR=${DEFAULT_INSTALLER_ASSETS_DIR:-$(pwd)}
 KUBEADMIN_USER=${KUBEADMIN_USER:-"kubeadmin"}
 KUBEADMIN_PASSWORD_FILE=${KUBEADMIN_PASSWORD_FILE:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeadmin-password"}
+
+# Registry.redhat.io username and password for local testing
+REGISTRY_UN=""
+REGISTRY_PASS=""
+
```

Comment:
Can you please execute just ```env``` in this script to check what are the environment variable available/injected in the test container.